openapi-maven-plugin/src/main/java/io/github/kbuntrock/DocumentationMojo.java

```diff
@@ -119,6 +116,7 @@ private void validateConfiguration() throws MojoFailureException {
 		}
 		this.getApiConfiguration().initDefaultValues();
 		EnumConfigHolder.storeConfig(this.getApiConfiguration().getEnumConfigList());
+		NullableConfigurationHolder.storeConfig(this.getApiConfiguration());
```

Comment:
This comment should be read after the one on "CommonApiConfiguration" to be understood.

This line should not be called here, but instead just after this line :
"final ApiConfiguration apiConfig = initialApiConfiguration.mergeWithCommonApiConfiguration(this.apiConfiguration);"
(in the scanProjectResourcesAndWriteSpec function)

Therefore, it can be overridden by any ApiConfiguration.